docs/high_availability_installation_guide.md

```diff
@@ -164,36 +159,102 @@ run the follow command to apply the change.
+##### 3> cd to ```harbor``` directory 
+##### 4> Configure hostname
+In ```harbor.cfg```
+
+```
+hostname = reg.mydomain.com 
+
+```
+Change reg.mydomain.com to your FQDN or VIP(eg. 192.168.111.220)
+
+##### 5> Provide the Harbor db connection info
+
+In ```Harbor.cfg```
+Change the value to your Harbor DB's
+
+```
+#The password for the root user of mysql db, change this before any production use.
+db_password = root123
+
+...
+
+#####################################################
+#the address of the mysql database.
+db_host = 192.168.1.215
+
+#The port of mysql database host
+db_port = 3306
+
+#The user name of mysql database
+db_user = root
+
+```
+
+##### 6> Provide the Redis server/cluster address
+In harbor.cfg
+
+```
+#The redis server address
+redis_url = 192.168.1.217:6379
+```
+
+If you have multiple Redis servers then separate them with ";"
+
+##### 7> Provide the Clair DB connection information
+In harbor.cfg
+
+```
+clair_db_password = password
+
+...
+
+#Clair DB host address
+clair_db_host = 192.168.1.50
+
+#Clair DB connect port
+clair_db_port = 5432
+
+#Clair DB username
+clair_db_username = postgres
+
+#Clair default database
+clair_db = postgres
+
+```
+##### 8> Provide the storage config information
+In harbor.cfg
+
+```
+### Docker Registry setting ###
+#registry_storage_provider can be: filesystem, s3, gcs, azure, etc.
+registry_storage_provider_name = filesystem
+#registry_storage_provider_config is a comma separated "key: value" pairs, e.g. "key1: value, key2: value2".
+#Refer to https://docs.docker.com/registry/configuration/#storage for all available configuration.
+registry_storage_provider_config =
+```
+
 You can find the configure examples for all kinds of storages at https://docs.docker.com/registry/configuration/#storage .  
 For example if you use swift as the storage backend then you need to set the follow:
 
 ```
 registry_storage_provider_name = swift
-registry_storage_provider_config = username: yourusername,password: yourpass,authurl: http://192.168.1.217/identity/v3,tenant: admin,domain: default,region: RegionOne,container: docker_images
+registry_storage_provider_config = username:yourusername,password:yourpass,authurl:http://192.168.1.217/identity/v3,tenant:admin,domain:default,region:RegionOne,container:docker_images
 ```
 
-**Important!**
+**Important**
 
 If you set ```filesystem``` for the ```registry_storage_provider_name``` you must make sure the registry directory ```/data/registry``` mount to a shared storage like NFS,Ceph, etc. You need to create the /data/registry directory first and change it's owner to 10000:10000 as registry will run as userID 10000 and groupID 10000. 
 
- - 13> Import Harbor database schema to your external MariaDB 
+##### 9> Import Harbor database schema to your external MariaDB 
 ```
     Copy the ~/harbor/ha/registry.sql file to a machine that has MariaDB client installed 
```

Comment:
Could we move this step up to the part of MariaDB? It's so confuse to put this step here because it's not a setting of Harbor node.

Let user to get the registry.sql from source and then import it with Maria Client.